docs/compress-options.md

```diff
@@ -69,6 +69,14 @@ Default: `false`
 
 Pretty print file sizes when logging.
 
+## createEmptyArchive
```

Comment:
I am not sure about header level. What do you think?

I ask becase all options in README.md file has h4, but in compress-options.md some has h3 and some has h2. Why so and what header is correct there?